### PR TITLE
[BEAM-14428] I/O, community, and contribute pages improvements

### PR DESCRIPTION
**Pull request by bullet03**
**Please** add a meaningful description for your change here
[BEAM-14428]
- change text of nav contribure and community nav menus
- change styling of Ways You Can Contribute table
- change styling of Contact Us table

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @nausharipov

File: website/www/site/assets/scss/_contribute.sass [hunk 1]
```diff
@@ -39,3 +40,54 @@
...
+      &:hover,
+      &focus
+        text-decoration: underline
+
+  td:nth-child(1)
+    background-color: rgba(255, 109, 0, 0.04)
+    padding-left: $padding/2 !important
+    padding-right: 0 !important
+    line-height: 1.57
+    letter-spacing: 0.43px
```
Comment (nausharipov) [inline website/www/site/assets/scss/_contribute.sass:93]: consider storing 0.43px in a variable

**Comment by dariabezkorovaina** [comment]
@pcoet can you please review this PR? There are some minor style and consistency improvements we discussed with Brittany and Danielle :)

**Review by pcoet — APPROVED** [review]

**Comment by aromanenko-dev** [comment]
Is it ready to be merged?

**Comment by nausharipov** [comment]
> Is it ready to be merged?

yes

**Comment by nausharipov** [comment]
> 

yes

**Comment by dariabezkorovaina** [comment]
@aromanenko-dev can you please help us merge this one?:)

**Comment by aromanenko-dev** [comment]
@dariabezkorovaina Sure!

**Pull request merged**